docs-source/spring/content/development/cli/_index.md

```diff
@@ -181,41 +214,47 @@ The following is an example development workflow using the CLI:
    obaas-cli [delete]: Application/Namespace [myapp] as successfully deleted
    ```
 
-4. Use the `bind` command to create and update a database schema or user for the service. These commands also create or update the Kubernetes secret and binding environment entries for the schema. These are set in the Kubernetes deployment created with the `deploy` command. For example:
+### bind
 
-   ```cmd
-   oractl:>help bind
-   NAME
-          bind - Create a schema/user and bind it to service deployment.
+Use the `bind` command to create and update a database schema or user for the service. These commands also create or update the Kubernetes secret and binding environment entries for the schema. These are set in the Kubernetes deployment created with the `deploy` command. For example:
 
-   SYNOPSIS
-          bind --action CommandConstants.BindActions --app-name String --service-name String --username String --binding-prefix String --help
+```cmd
+oractl:>help bind
+NAME
+       bind - Create or Update a schema/user and bind it to service deployment.
+
+SYNOPSIS
+       bind --action CommandConstants.BindActions --app-name String --service-name String --username String --binding-prefix String --help
+
+OPTIONS
+       --action CommandConstants.BindActions
+       possible actions: create or update. create is default.
+       [Optional, default = create]
 
-   OPTIONS
-          --action CommandConstants.BindActions
-          possible actions: create or update. create is default.
-          [Optional, default = create]
+       --app-name String
+       application/namespace
+       [Optional, default = application]
 
-          --app-name String
-          application/namespace
-          [Optional, default = application]
+       --service-name String
+       Service Name (Default for database user if username is not provided)
+       [Optional]
 
-          --service-name String
-          Service Name/Database User
-          [Mandatory]
+       --username String
+       Database User
+       [Optional]
 
-          --username String
-          Database User
-          [Optional]
+       --binding-prefix String
+       spring binding prefix
+       [Optional, default = spring.datasource]
 
-          --binding-prefix String
-          spring binding prefix
-          [Optional, default = spring.datasource]
+       --help or -h
+       help for bind
+       [Optional]
 
-          --help or -h
-          help for bind
-          [Optional]
-   ```
+Ask for Help
+       Slack: https://oracledevs.slack.com/archives/C03ALDSV272
+       E-mail: obaas_ww@oracle.com
+```
 
    > ATTENTION: The `service-name` is mandatory and used as the name for the Schema/User to be created. If you want to use a different Schema/User from the `service-name`, you must also submit the`username`.
```

Comment:
Change both instances of "Schema/User" to "schema or user"

In the last sentence, add a space between "the 'username'."